odbc/src/api/functions.rs

```diff
@@ -32,6 +34,23 @@ const HANDLE_MUST_BE_CONN_ERROR: &str = "handle must be conn";
 const HANDLE_MUST_BE_STMT_ERROR: &str = "handle must be stmt";
 const HANDLE_MUST_BE_DESC_ERROR: &str = "handle must be desc";
 
+// Verifies that the expected SQL State, message text, and native error in the handle match
```

Comment:
Sorry if the comment comes from me (probably), but I don't think that it describe what the function does. Can you update it?

Reply:
Sure, updated